plugin/pkg/auth/authenticator/token/tokenfile/tokenfile.go

```diff
@@ -40,6 +40,8 @@ func NewCSV(path string) (*TokenAuthenticator, error) {
 
 	tokens := make(map[string]*user.DefaultInfo)
 	reader := csv.NewReader(file)
+	reader.FieldsPerRecord = -1
+	reader.Comment = '#'
```

Comment:
not sure we want to enable comments in these files... this doesn't seem related to this PR